src/modules/cmdpal/Exts/Microsoft.CmdPal.Ext.System/Helpers/Icons.cs

```diff
@@ -8,11 +8,11 @@ namespace Microsoft.CmdPal.Ext.System.Helpers;
 
 public static partial class Icons
 {
-    public static IconInfo FirmwareSettingsIcon { get; } = IconHelpers.FromRelativePaths("Microsoft.CmdPal.Ext.System\\Assets\\logoff.light.png", "Microsoft.CmdPal.Ext.System\\Assets\\logoff.dark.png");
+    public static IconInfo FirmwareSettingsIcon { get; } = new IconInfo("\uEBC4");
```

Comment:
okay so this replaces 
![image](https://github.com/user-attachments/assets/afe0046f-b50d-4158-859e-7a9b2d419aa8)

with 

![image](https://github.com/user-attachments/assets/4ae86735-fd2c-403f-9dc0-bce05eecd582)

Reply:
Replaced it with: 

![image](https://github.com/user-attachments/assets/2bfe7a0f-42e0-455f-b056-974ad6325934)